net/lighttpd/Makefile

```diff
@@ -41,13 +86,15 @@ endef
 define Package/lighttpd
   $(call Package/lighttpd/Default)
   MENU:=1
-  DEPENDS:=+libnettle +libpcre2 +libpthread +LIGHTTPD_LOGROTATE:logrotate
+  DEPENDS:=+libpthread +LIGHTTPD_LOGROTATE:logrotate \
+           $(if $(CONFIG_LIGHTTPD_PCRE2),+libpcre2,) \
```

Comment:
why not use `+LIGHTTPD_PCRE2:libpcre2` here?

Reply:
Will do.  I think I misunderstood the package caching from `make package/index` (before some of my make syntax can execute with config results) and make evaluation at build time.